pkgs/applications/networking/instant-messengers/franz/generic.nix

```
@@ -66,12 +76,33 @@ stdenv.mkDerivation rec {
     cups
     expat
     stdenv.cc.cc
+    wayland
+    pipewire
   ];
+
   runtimeDependencies = [ stdenv.cc.cc.lib (lib.getLib udev) libnotify libappindicator-gtk3 ];
+  libPath = lib.makeLibraryPath buildInputs
+    + lib.optionalString (stdenv.is64bit)
+    (":" + lib.makeSearchPathOutput "lib" "lib64" buildInputs)
```

```suggestion
    + lib.optionalString stdenv.is64bit (":" + lib.makeSearchPathOutput "lib" "lib64" buildInputs)
```
otherwise the formatting is confusing